Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -500,6 +501,8 @@ static inline void continueAfterRequest(RetainPtr<id <ASCCredentialProtocol>> cr
     }
 #endif // PLATFORM(MAC) || PLATFORM(MACCATALYST)
 #if PLATFORM(IOS)
```

Comment:
@pascoej Is this `PLATFORM(IOS)` intentionally? Do we want to exclude Catalyst here?

Reply:
Yes this is intentional. Catalyst is covered in the condition above.